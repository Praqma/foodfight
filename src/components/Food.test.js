import React from 'react';
import Food from './Food';
import renderer from 'react-test-renderer';

test('renders the name', () => {
    const component  = renderer.create(
        <Food name='Potato'/>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})  