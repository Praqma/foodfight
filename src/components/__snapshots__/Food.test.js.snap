// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the name 1`] = `
<p>
  Potato
</p>
`;
